ahnlich/ai/src/manager/mod.rs

```diff
@@ -312,7 +312,7 @@ impl ModelManager {
 
         let (response_tx, response_rx) = oneshot::channel();
         let request = ModelThreadRequest {
-            inputs,
+            inputs: inputs.as_ref().clone(),
```

Comment:
Do we need `.as_ref()`.. should be able to clone the arc itself with `Arc::clone`

Reply:
Might need modifying ModelThreadInput though

Reply:
I'd let you decide.... if at it's very root, we cannot ship the `Arc<Vec<StoreInput>>` to the models for inference without cloning, then perhaps we just get rid of the `Arc` all together and clone .... but feel free to change it all the way down if it works

It won't affect the SDKs at all as this is internal to just AI proxy